### ISSUE TITLE
Terraform plans is only available within a single phase 

### ISSUE THREAD

**Issue by arielrolfo** [chef-boneyard/delivery-sugar#36]
Hi gurus,

I've been playing around this: https://github.com/chef-cookbooks/delivery-sugar/blob/master/examples/terraform/.delivery/build_cookbook/README_TERRAFORM.md  but I cannot use it other than in a  single phase (like provision), it feels awkward provision the VM, bootstrapping it and running the tests in the same phase.
We're currently able to use a REST HTTP backend to store the terraform state, so wondering how that might be leveraged to spread the execution across all phases in Acceptance stage.

**Comment by mattlqx** [comment]
Agreed. Without a deprovision phase, it's hard to cleanup any ephemeral resources created by the provision phase that would be deployed to and tested in subsequent phases. I don't really understand the implementation here.

**Comment by cgunasree08** [comment]
This issue or pull request is being automatically closed because the repository is being archived as part of the [Repository Standardization Initiative](https://github.com/chef-boneyard/oss-repo-standardization-2025).
